packages/less/Gruntfile.js

```diff
@@ -230,6 +230,7 @@ module.exports = function(grunt) {
             },
             test: {
                 command: [
+                    "TS_NODE_SCOPE=true",
```

Comment:
Add the special ENV for CI , https://github.com/TypeStrong/ts-node/issues/693#issuecomment-848948517 for more details.